cmd/deployment_test.go

```diff
@@ -80,9 +80,23 @@ func TestDeploymentCreateCommand(t *testing.T) {
 	})
 	api := houston.NewHoustonClient(client)
 
-	_, output, err := executeCommandC(api, "deployment", "create", "new-deployment-name", "--executor=celery")
-	assert.NoError(t, err)
-	assert.Contains(t, output, "Successfully created deployment with Celery executor. Deployment can be accessed at the following URLs")
+	myTests := []struct {
+		cmdArgs        []string
+		expectedOutput string
+		expectedError  string
+	}{
+		{cmdArgs: []string{"deployment", "create", "new-deployment-name", "--executor=celery", "--dag-deployment-type=volume", "--nfs-location=test:/test"}, expectedOutput: "", expectedError: "unknown flag: --dag-deployment-type"},
```

Comment:
Dont you want to say something like unsupported option when server does not have DAGDeployment option configured to be true?

Reply:
not sure that understand you.

Reply:
you are showing an error "unknown flag: --dag-deployment-type" when dag-deployment-type is specified in the command line when ConfigureDAGDeployment  feature flag is set to false. Correct?

I am saying that the error should be something like "Unsupported option dag-deployment-type when ConfigureDAGDeployment is disabled.

Reply:
we can't change this text, it's default text of cobra, so we enable NFS related options only when houston enable this feature otherwise we hide options and examples.

Reply:
here is an example:
```
astro --test
Error: unknown flag: --test
Usage:
  astro [command]

Available Commands:
  auth            Authenticate with an Astronomer Cluster
  cluster         Manage Astronomer Clusters
  completion      Generate autocompletions script for the specified shell (bash or zsh)
  config          Manage project configuration
  deploy          Deploy an Airflow project
  deployment      Manage Astronomer Deployments
  dev             Manage Airflow projects
  help            Help about any command
  upgrade         Check for newer version of Astronomer CLI
  user            Manage Astronomer user
  version         astro CLI version
  workspace       Manage Astronomer Workspaces

Flags:
  -h, --help                 help for astro
```

Reply:
ok